actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -199,6 +199,42 @@ def link_to(name = nil, options = nil, html_options = nil, &block)
         content_tag("a".freeze, name || url, html_options, &block)
       end
 
+      # Returns an array of hashes each containing :name and :value keys
+      # suitable for use as the names and values of form input fields:
+      #
+      #   to_form_params({name: 'David', nationality: 'Danish'})
+      #   # => [{name: :name, value: 'David'}, {name: 'nationality', value: 'Danish'}]
+      #
+      #   to_form_params({country: {name: 'Denmark'})
+      #   # => [{name: 'country[name]', value: 'Denmark'}]
+      #
+      #   to_form_params({countries: ['Denmark', 'Sweden']})
+      #   # => [{name: 'countries[]', value: 'Denmark'}, {name: 'countries[]', value: 'Sweden'}]
+      #
+      # An optional namespace can be passed to enclose key names:
+      #
+      #   to_form_params({name: 'Denmark'}, 'country')
+      #   # => [{name: 'country[name]', value: 'Denmark'}]
+      def to_form_params(attribute, namespace = nil) # :nodoc:
```

Comment:
Yes. This method should not be used by users (this is what `# :nodoc:` is for) and it is only used in this file (so it has private visibility)